test/unit/common/CommonUtils.test.ts

```diff
@@ -6,20 +6,73 @@
  */
 import os from 'node:os';
 import fs from 'node:fs';
+import http from 'node:http';
+import https from 'node:https';
 import path from 'node:path';
-import { Messages } from '@salesforce/core';
+import stream from 'node:stream';
+import * as childProcess from 'node:child_process';
+import { Logger, Messages } from '@salesforce/core';
 import { TestContext } from '@salesforce/core/testSetup';
 import { stubMethod } from '@salesforce/ts-sinon';
 import { expect } from 'chai';
+import sinon from 'sinon';
 import { CommonUtils } from '../../../src/common/CommonUtils.js';
 
 Messages.importMessagesDirectoryFromMetaUrl(import.meta.url);
 
+// Custom Writable stream that writes to memory, for testing downloadFile
+const data: Buffer[] = [];
+class MemoryWriteStream extends stream.Writable {
+    public getData(): string {
+        return Buffer.concat(data).toString();
+    }
+
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    public _write(chunk: Buffer, encoding: string, callback: (error?: Error | null) => void): void {
+        data.push(chunk);
+        callback();
+    }
+
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    public close(callback?: ((err?: NodeJS.ErrnoException | null | undefined) => void) | undefined): void {
```

Comment:
Do the overrides need to have their parent's class's method?

Reply:
Good point, no they don't.